Index: xscreensaver/driver/XScreenSaver.ad.in
===================================================================
--- xscreensaver.orig/driver/XScreenSaver.ad.in	2014-10-07 22:51:11.000000000 +0200
+++ xscreensaver/driver/XScreenSaver.ad.in	2014-10-07 22:51:11.000000000 +0200
@@ -195,6 +195,7 @@
 				goop -root				    \n\
 				grav -root				    \n\
 				ifs -root				    \n\
+				unicode -root				    \n\
 @GL_KLUDGE@ GL: 				jigsaw -root				    \n\
 				julia -root				    \n\
 -				kaleidescope -root			    \n\
