Index: xscreensaver/driver/XScreenSaver.ad.in
===================================================================
--- xscreensaver.orig/driver/XScreenSaver.ad.in	2014-10-07 22:51:38.000000000 +0200
+++ xscreensaver/driver/XScreenSaver.ad.in	2014-10-07 22:51:38.000000000 +0200
@@ -424,10 +424,11 @@
 *Dialog.shadowThickness:	2
 
 *passwd.heading.label:		XScreenSaver %s
-*passwd.body.label:		This screen is locked.
+*passwd.body.label:		Please enter your password.
 *passwd.unlock.label:		OK
 *passwd.login.label:		New Login
 *passwd.user.label:		Username:
+*passwd.passwd.label:		Password:
 *passwd.thermometer.width:	8
 *passwd.asterisks:              True
 *passwd.uname:                  True
