#
# Prompts more consistent with the Debian experience (?)
#
Index: xscreensaver-git/driver/XScreenSaver.ad.in
===================================================================
--- xscreensaver-git.orig/driver/XScreenSaver.ad.in	2016-12-11 14:23:39.828291589 +0100
+++ xscreensaver-git/driver/XScreenSaver.ad.in	2016-12-11 14:23:39.824291587 +0100
@@ -440,10 +440,11 @@
 *Dialog.shadowThickness:	2
 
 *passwd.heading.label:		XScreenSaver %s
-*passwd.body.label:		This screen is locked.
+*passwd.body.label:		Please enter your password.
 *passwd.unlock.label:		OK
 *passwd.login.label:		New Login
 *passwd.user.label:		Username:
+*passwd.passwd.label:		Password:
 *passwd.thermometer.width:	8
 *passwd.asterisks:              True
 *passwd.uname:                  True
