Index: xscreensaver-git/Makefile.in
===================================================================
--- xscreensaver-git.orig/Makefile.in	2016-12-11 15:04:10.000000000 +0100
+++ xscreensaver-git/Makefile.in	2016-12-11 15:05:25.734072018 +0100
@@ -7,7 +7,7 @@
 
 SHELL		= /bin/sh
 SUBDIRS 	= utils jwxyz hacks hacks/glx driver po
-SUBDIRS2	= $(SUBDIRS) OSX android
+SUBDIRS2	= $(SUBDIRS)
 TARFILES	= README README.hacking README.VMS INSTALL \
 		  configure configure.in Makefile.in config.h.in \
 		  config.h-vms install-sh setup.com config.guess aclocal.m4 \
