#
# For reproducible builds. Bug #819595
#
Index: xscreensaver-git/hacks/Makefile.in
===================================================================
--- xscreensaver-git.orig/hacks/Makefile.in	2016-12-11 14:23:49.948297205 +0100
+++ xscreensaver-git/hacks/Makefile.in	2016-12-11 14:23:49.948297205 +0100
@@ -853,8 +853,8 @@
 
 m6502.h:
 	@echo "building m6502.h from $(srcdir)/images/m6502/*.asm"; \
-	UTILS_SRC="$(UTILS_SRC)" \
-	$(srcdir)/m6502.sh m6502.h $(srcdir)/images/m6502/*.asm
+	find $(srcdir)/images/m6502/ -name '*.asm' | LC_ALL=C sort | \
+	  UTILS_SRC="$(UTILS_SRC)" xargs $(srcdir)/m6502.sh m6502.h
 
 m6502.o:	m6502.h
 m6502:		m6502.o		asm6502.o $(HACK_OBJS) $(ATV)
