#
# This "please upgrade" message would drive users crazy. Bug #819703
#
Index: xscreensaver-git/driver/prefs.c
===================================================================
--- xscreensaver-git.orig/driver/prefs.c	2016-05-28 19:06:12.441676714 +0200
+++ xscreensaver-git/driver/prefs.c	2016-05-28 19:06:12.437676714 +0200
@@ -1736,6 +1736,8 @@
      everyone involved, most especially the users.
   */
 
+  return 0;
+
   time_t now = time ((time_t *) 0);				/*       */
   struct tm *tm = localtime (&now);				/*   d   */
   const char *s = screensaver_id;				/*   o   */
